generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-cxf-soap-grouped/pom.xml

```diff
@@ -22,6 +22,17 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.quarkiverse.cxf</groupId>
+      <artifactId>quarkus-cxf-test-util</artifactId>
+      <version>3.20.0.CR1</version>
```

Comment:
@ppalaga this is a bit odd? `CR1`?

It's probably not blocking as AFAICS, it's only in tests.

Reply:
Yeah, it is because CQ 3.20.0 was released before QCXF 3.20.0. 
It's fine, because it is only in a test.